gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -583,3 +592,45 @@ def add_arguments(self, parser):
               'be the same as the BigQuery table, but it requires all '
               'extracted variants to have the same call name ordering (usually '
               'true for tables from single VCF file import).'))
+
+
+def _validate_inputs(parsed_args):
+  if ((parsed_args.input_pattern and parsed_args.input_file) or
+      (not parsed_args.input_pattern and not parsed_args.input_file)):
+    raise ValueError('One and only one of input_pattern and input_file has '
```

Comment:
Usually written as "Exactly one"

"Exactly one of input_pattern or input_file must be provided"